service/src/main/resources/application.yml

```diff
@@ -205,8 +208,6 @@ landingzone:
 feature:
   # azure-enabled - Controls inclusion of Azure support in WSM
   azure-enabled: true
-  # aws-enabled - Controls inclusion of AWS support in WSM
-  aws-enabled: false
```

Comment:
Can we keen aws-enabled in place till feature service is set up and tested in all env?
i.e. keep the plumbing for aws-enabled as is, but replace the call to isAwsEnabled with the call to feature service

Reply:
i set the default value to false in the FeatureService.

Reply:
Sure, no concern if tested